lighthouse-core/audits/is-on-https.js

```diff
@@ -17,7 +17,9 @@ const UIStrings = {
   failureTitle: 'Does not use HTTPS',
   /** Description of a Lighthouse audit that tells the user *why* HTTPS use is important. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
```

Comment:
```suggestion
  /** Description of a Lighthouse audit that tells the user *why* HTTPS use *for all resources* is important. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
```

just some extra emphasis for the translators to keep the mixed content bit important perhaps?